app/src/userpages/components/StreamPage/List/index.jsx

```diff
@@ -51,26 +47,341 @@ import Sidebar from '$shared/components/Sidebar'
 import SidebarProvider, { SidebarContext } from '$shared/components/Sidebar/SidebarProvider'
 import ShareSidebar from '$userpages/components/ShareSidebar'
 import { ago } from '$shared/utils/time'
-import { subscribeSnippets } from '$utils/streamSnippets'
-
-import styles from './streamsList.pcss'
-
-const { lg } = breakpoints
+import { MD, LG } from '$shared/utils/styled'
+import useModal from '$shared/hooks/useModal'
+import SnippetDialog from './SnippetDialog'
+
+const DesktopOnlyButton = styled(Button)`
+    && {
+        display: none;
+    }
+
+    @media (min-width: ${LG}px) {
+        && {
+            display: inline-flex;
+        }
+    }
+`
 
 export const CreateStreamButton = () => (
-    <Button
-        className={styles.createStreamButton}
+    <DesktopOnlyButton
         tag={Link}
         to={routes.streams.new()}
     >
         <Translate value="userpages.streams.createStream" />
-    </Button>
+    </DesktopOnlyButton>
 )
 
-const Dialogs = {
-    SHARE: 'share',
-    SNIPPET: 'snippet',
-}
+const StreamTable = styled.div`
+    @media (min-width: ${MD}px) {
+        margin-top: 50px;
+        border-top: 1px solid #CDCDCD;
+    }
+
+    @media (min-width: ${LG}px) {
+        margin-top: 0;
+        margin-bottom: auto;
+        border-top: none;
+    }
+`
+
+const StyledListContainer = styled(ListContainer)`
+    && {
+        padding: 0;
+        margin-bottom: 4em;
+    }
+
+    ${StreamTable}:last-child {
+        margin-bottom: 4rem;
+    }
+
+    @media (min-width: ${MD}px) {
+        && {
+            padding-left: 1.5rem;
+            padding-right: 1.5rem;
+        }
+    }
+
+    @media (min-width: ${LG}px) {
+        && {
+            margin-bottom: 0;
+        }
+    }
+`
+
+const LoadingMoreSpacer = styled.div`
+    margin-top: 11.5rem;
+
+    @media (min-width: ${MD}px) {
+        margin-top: 15.5rem;
+    }
+
+    @media (min-width: ${LG}px) {
+        margin-top: 7.5rem;
+    }
+`
+
+const Row = styled.div`
+    display: grid;
+    grid-row-gap: 0rem;
+    grid-column-gap: 1rem;
+    border-bottom: 1px solid #CDCDCD;
+    padding: 1rem 1.5rem 1.5rem;
+    grid-template-columns: minmax(0, 1fr) 12px;
+    align-items: center;
+
+    @media (min-width: ${MD}px) {
+        grid-template-columns: minmax(0, 1fr) 184px 12px;
+    }
+
+    @media (min-width: ${LG}px) {
+        grid-template-columns: minmax(200px, 3fr) 3fr minmax(136px, 2fr) minmax(136px, 2fr) minmax(68px, 1fr) 32px;
+        padding: 1rem 0.75rem;
+    }
+`
+
+const HeaderRow = styled(Row)`
+    display: none;
+
+    @media (min-width: ${LG}px) {
+        display: grid;
+        padding: 0.75rem;
+        line-height: 14px;
+    }
+`
+
+const TableRow = styled(Row)`
+    cursor: pointer;
+
+    &:hover,
+    &:focus-within {
+        background-color: #EFEFEF;
+
+        .dropdown {
+            visibility: visible;
+        }
+    }
+`
+
+const RowItem = styled.div`
+    display: none;
+    overflow-y: visible;
```

Comment:
Selective `overflow` means potential trouble in the future. What problem does it solve?

Reply:
I think this is no longer needed